### PR TITLE
Update `Account`

### PR DESCRIPTION
**Pull request by Furdox**
Added the fact that usernames cannot be taken away if that account has any ranked beatmaps. (A user on Reddit warned me that support told them that.)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Furdox** [comment]
in the request, there are unrelated commits about an owc translation but was really barebones and I just didn't want to translate it, so it gets deleted. Doesn't actually affect the final result.

**Comment by peppy** [comment]
@Ephemeralis please confirm this is some unwritten rule I've never heard of.

**Comment by Furdox** [comment]
@peppy If that rule doesn't exist, it could be created to make less confusion. </-- >w^ -->

**Comment by cl8n** [comment]
osu-web won't let a name be taken if it has badges or ranked maps. <https://github.com/ppy/osu-web/blob/851eb313e6c1ac7a3de58799a13ae0f6f58a4dac/app/Libraries/UsernameValidation.php#L81-L112>

**Comment by Furdox** [comment]
@cl8n That means I could also add the badge requirement, I'll go do that.

**Comment by Furdox** [comment]
Wait. If I just say "user badges", people will get confused. What's "badge" exactly? The little banners right below the name for winning tournaments?

**Comment by Furdox** [comment]
I'll go to bed. I'll probably check the answer tommorow and edit the commit.

**Comment by Furdox** [comment]
Dang, still no answer.

**Comment by Walavouchey** [comment]
> Wait. If I just say "user badges", people will get confused. What's "badge" exactly? The little banners right below the name for winning tournaments?

"profile badge" https://osu.ppy.sh/wiki/fr/Community/Profile_badge

**Comment by Furdox** [comment]
> > Wait. If I just say "user badges", people will get confused. What's "badge" exactly? The little banners right below the name for winning tournaments?
> 
> "profile badge" https://osu.ppy.sh/wiki/fr/Community/Profile_badge

oh ok, making the changes.

**Comment by Furdox** [comment]
Updated commits.

File: wiki/Help_centre/Account/en.md [hunk 1]
```diff
@@ -150,6 +150,8 @@ If a player has not signed in for at least 6 months and has no plays, their user
 
 If that player **does** have any plays across all game modes, a [non-linear function](https://www.desmos.com/calculator/b89siyv9j8) is used to calculate how much extra time is added to those 6 months. This is to prevent people from stealing the usernames of active or recently retired players.
 
+Note that players with [ranked beatmaps](https://osu.ppy.sh/wiki/en/Beatmap/Category#ranked) or [user badges](https://osu.ppy.sh/wiki/en/Community/Profile_badge) may not be eligible.
```
Comment (Walavouchey) [inline wiki/Help_centre/Account/en.md:153]:
```suggestion
Note that usernames of players with [ranked beatmaps](https://osu.ppy.sh/wiki/en/Beatmap/Category#ranked) or [profile badges](https://osu.ppy.sh/wiki/en/Community/Profile_badge) cannot be taken.
```

**Comment by Furdox** [comment]
wait, accidental commit. wanted to add a comment to a suggestion but github moved to the top of my phone, pressing the accept button.

**Comment by Furdox** [comment]
I can't revert the commit...

**Comment by Furdox** [comment]
welp, I guess it's gonna be formal. lemme do the translation and fix the english link.

**Comment by Furdox** [comment]
Done.

**Comment by Furdox** [comment]
The links still somehow fails some check, will need a reviewer.

**Review by Walavouchey — COMMENTED** [review]
needs check from eph, otherwise ok

**Comment by Furdox** [comment]
Thanks for the revision! And now, we play the waiting game...

**Review by peppy — APPROVED** [review]

**Pull request merged**